sdk/formrecognizer/ai-form-recognizer/src/utils/utils.browser.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type, @typescript-eslint/no-empty-function
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type, @typescript-eslint/no-empty-function, @typescript-eslint/explicit-module-boundary-types
```

Comment:
seems like we could stub this method better? even if took and returned `unknown` to avoid referencing stream types